src/collections/Events/constants.ts

```diff
@@ -54,61 +44,4 @@ export const eventTypesData: EventType[] = [
   },
 ]
 
-interface QuickDateFilter {
-  id: string
-  label: string
-  startDate: () => string // Returns ISO date string
-  endDate: () => string | null // Returns ISO date string or null for "no end"
-  sortDirection: 'asc' | 'desc' // How events should be sorted for this filter
-}
-
-export const QUICK_DATE_FILTERS: QuickDateFilter[] = (() => {
-  const fmt = (date: Date) => format(date, 'MM-dd-yyyy')
-  const today = () => new Date()
-  const yesterday = () => subDays(today(), 1)
-
-  return [
-    {
-      id: 'upcoming',
-      label: 'Upcoming',
-      startDate: () => fmt(today()),
-      endDate: () => null,
-      sortDirection: 'asc',
-    },
-    {
-      id: 'this-week',
-      label: 'This Week',
-      startDate: () => fmt(today()),
-      endDate: () => fmt(endOfWeek(today(), { weekStartsOn: 0 })),
-      sortDirection: 'asc',
-    },
-    {
-      id: 'next-week',
-      label: 'Next Week',
-      startDate: () => fmt(addDays(startOfWeek(today(), { weekStartsOn: 0 }), 7)),
-      endDate: () => fmt(endOfWeek(addDays(today(), 7), { weekStartsOn: 0 })),
-      sortDirection: 'asc',
-    },
-    {
-      id: 'this-month',
-      label: 'This Month',
-      startDate: () => fmt(today()),
-      endDate: () => fmt(endOfMonth(today())),
-      sortDirection: 'asc',
-    },
-    {
-      id: 'next-month',
-      label: 'Next Month',
-      startDate: () => fmt(startOfMonth(addMonths(today(), 1))),
-      endDate: () => fmt(endOfMonth(addMonths(today(), 1))),
-      sortDirection: 'asc',
-    },
-    {
-      id: 'past-events',
-      label: 'Past Events',
-      startDate: () => fmt(subYears(today(), 1)),
-      endDate: () => fmt(yesterday()),
-      sortDirection: 'desc',
-    },
-  ]
-})()
+export const QUICK_DATE_FILTERS = QUICK_DATE_FILTERS_EVENTS
```

Comment:
Can we just assign the filters here rather than in the `quickDateFilters` file? Or even just define it in `src/app/(frontend)/[center]/events/events-date-filter.tsx` similarly to how I suggested we assign this for courses, but I think it is used in other files.

```suggestion
export const QUICK_DATE_FILTERS = createQuickDateFilters('Past Events')
```